cmd/fund/usage.md

```diff
@@ -1,10 +1,51 @@
+Bulk fund crypto wallets automatically.
+
 ```bash
-$ polycli fund \
-  --rpc-url="https://rootchain-devnetsub.zkevmdev.net"  \
-  --funding-wallet-pk="REPLACE" \
-  --wallet-count=5 \
-  --wallet-funding-amt=0.00015 \
-  --wallet-funding-gas=50000 \
-  --concurrency=5 \
-  --output-file="/opt/funded_wallets.json"
-```
\ No newline at end of file
+# Fund wallets specified by the user.
+$ polycli fund --addresses 0x5eD3BE7a1cDafd558F88a673345889dC75837aA2,0x1Ec6efdBd371D6444779eAE7B7e16907e0c8eC27
+3:43PM INF Starting bulk funding wallets
+3:43PM INF Using addresses provided by the user
+3:43PM INF Wallet(s) funded! 💸
+3:43PM INF Total execution time: 1.020693583s
+
+# Fund 20 wallets, derived from the default private key.
+$ polycli fund --number 20 --hd-derivation true
+3:44PM INF Starting bulk funding wallets
+3:44PM INF Deriving addresses from the default mnemonic
+Error: todo: implement this mode
```

Comment:
do we have follow-on ticket for this piece?

Reply:
I already implemented yet, I just need to update `usage.md`!